src/magellan_firmware/firmware/magellan_controller/config.h

```diff
@@ -26,19 +26,23 @@ extern RobotState currentState;
 #define SERVO_PWM 4
 
 // Steering trim
-#define STEERING_OFFSET -0.2
+#define STEERING_OFFSET 0
 
 #define THROTTLE_MIN 0.1
 #define STEERING_MIN 0.05
 
-#define LEFT_ENCODER 20
+#define ENCODER_A 6
+#define ENCODER_B 7
+#define ENCODER_C 8
+
 #define RUN_BUTTON 21
-#define RIGHT_ENCODER 22
 
+
+// TODO: Measure and change vars accordingly
 // Distance from front wheel to back wheel
-const constexpr double kTrackLength = 0.3175;
+const constexpr double kTrackLength = 0.39; // 39 cm
 // Distance from left wheel to right wheel
-const constexpr double kTrackWidth = 0.24;
+const constexpr double kTrackWidth = 0.31;  // 31 cm
```

Comment:
Just `constexpr` is fine here - I think the other ones in this file are from a bad copy/paste

Reply:
`super const`